web/skins/classic/views/js/montage.js

```diff
@@ -798,7 +798,7 @@ function initPage() {
 
   setInterval(() => { //Updating GridStack resizeToContent, Scale & Ratio
     if (changedMonitors.length > 0) {
-      changedMonitors.forEach(function(item, index, object) {
+      changedMonitors.slice().reverse().forEach(function(item, index, object) {
```

Comment:
Aha!  Excellent content for a comment.

So the slice makes a copy...  which we could use for on.  Maybe we should just use a while with popping off elements until the array is empty... so that if another item gets added we will process it.

Reply:
I'm not ready to say anything yet.